### ISSUE TITLE
File > Recent Projects and Solutions > Clear Projects List doesnt work

### ISSUE THREAD

**Issue by prasethu** [Microsoft/VS-PPT#166]
This command no longer works because starting 15.5 the File > Recent Projects menu uses the same data store that the Start Page uses. However, the command only clears the old MRU store that is no longer used.

**Comment by prnadago** [comment]
Confirmed this is a bug in the Power Commands extension.

**Comment by prasethu** [comment]
👍

**Comment by justcla** [comment]
+1 for Open sourcing Power Commands so users can fix this locally while waiting for an official fix.

**Comment by justcla** [comment]
@prasethu, Any chance you could advise the fix?

**Comment by Kaelum** [comment]
This has been an issue for a very long time, and I recently opened an issue for Visual Studio itself, as I didn't realize that it was a part of the Power Commands extension.  Here is the open issue on Visual Studio:

https://developercommunity.visualstudio.com/content/problem/180571/recent-projects-and-solutions-cant-be-cleared.html?childToView=193463#comment-193463

**Comment by Kaelum** [comment]
Since we're not getting anywhere here, and it is very unlikely that there will be a fix anytime soon, I decided to try and find a work-around.  If you go to the Visual Studio 2017 Start Page (**File** / **Start Page**), you will see a **Recent** list on the left.  That is the list of items that shows under **File** / **Recent Projects and Solutions**.  To remove an item, right click on it and select **Remove From List**.  This list itself is bugged, so if you can't right click on an entry, close the **Start Page**, and start again.

**Comment by justcla** [comment]
Good call. That's the [approach I recommend](https://github.com/Microsoft/VS-PPT/issues/167#issuecomment-362733844).

Thanks for sharing the detailed instructions, and the tip for restarting the Start Page when it bugs out.
